src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -195,6 +195,12 @@
      */
     public static final String MSG_ORDERING = "import.ordering";
 
+    /**
+     * A key is pointing to the warning message text in "messages.properties"
+     * file.
+     */
+    public static final String MSG_REDUNDANT_SEPARATION = "import.groups.separated.internally";
```

Comment:
redundant vs internally ...... you already do not know what you do.
Check should enforce no separation in group so it should be "separation in group".
MSG_REDUNDANT_SEPARATION -> MSG_SEPARATED_IN_GROUP
import.groups.separated.internally - > import.groups.separated.in.group